index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Odin Project</title>
+  </head>
+  <body>
+    <nav class="background header">
+      <ul>
+        <li>Header Logo</li>
+        <li>header link one</li>
+        <li>header link two</li>
+        <li>header link three</li>
+      </ul>
+    </nav>
+
+    <div class="">
+      <div class="">
+        <h1 class="hero">
+          This website is <br />
+          awesome
+        </h1>
+        <p class>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+          atque? Hic ea voluptate saepe distinctio nostrum quibusdam sed, quam
+          reprehenderit neque error reiciendis assumenda nulla enim explicabo
+          provident, obcaecati eligendi.
+        </p>
+
+        <button>
+            Sign up
+        </button>
+      </div>
+      <div class="col-6">
+      </div>
+    </div>
+
+
+    <div>
+        <h1>Some random information</h1>
+        <img src="#" alt="">
+        <p>this is some subtext under an illustration or image</p>
+
+        <img src="#" alt="">
+        <p>this is some subtext under an illustration or image</p>
+
+        <img src="#" alt="">
+        <p>this is some subtext under an illustration or image</p>
+
+        <img src="#" alt="">
+        <p>this is some subtext under an illustration or image</p>
+    </div>
```

Comment:
Please do not use empty img tags. It would be better if you add the source of the images instead of leaving blank or remove it if it not necessary.